.eslintrc.js

```diff
@@ -22,11 +23,47 @@ module.exports = {
   ],
 
   'rules': {
-    'indent': ['error', 2, { 'SwitchCase': 1 }],
-    'quotes': ['error', 'single', { 'allowTemplateLiterals': true }],
-    'semi': ['error', 'always'],
-    'comma-dangle': ['error', 'always-multiline'],
     'arrow-parens': ['error', 'always'],
     'no-console': 0,
+
+    // Stylistic rules (automatically fixed on commit).
+    'array-bracket-spacing': ['error', 'never'],
+    'brace-style': ['error', '1tbs', { 'allowSingleLine': true }],
```

Comment:
Ah yes, the theoretical bug that has never happened 🙄 

My two cents: Not allowing single-line bloats function body, especially if your function requires several different types of guard. I'll give an example.

With this style the guards take up minimal space allowing the reader to focus on what the function actually does:

```js
function myFunction() {
  if (foo) return;
  if (!bar) return;
  if (baz) return;

  // actual logic here ...
}
```

With this style the guards take up a total of 4 lines each (including line-break between blocks), 4x the space! File length balloons and readability suffers. As a reader I want to see what this function does first and then under what conditions it should execute.

```js
function myFunction() {
  if (foo) {
    return;
  }

  if (!bar) {
    return;
  }

  if (baz) {
    return;
  }

  // actual logic here ...
}
```

* steps down off soap box *

I've said my piece, not going to dwell on it further :)